rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -341,16 +336,16 @@ lv2_fs_object::lv2_fs_object(utils::serial& ar, bool)
 u64 lv2_file::op_read(const fs::file& file, vm::ptr<void> buf, u64 size)
 {
 	// Copy data from intermediate buffer (avoid passing vm pointer to a native API)
-	uchar local_buf[65536];
+	std::vector<uchar> local_buf(std::min<u64>(size, 65536));
 
 	u64 result = 0;
 
 	while (result < size)
 	{
-		const u64 block = std::min<u64>(size - result, sizeof(local_buf));
-		const u64 nread = file.read(+local_buf, block);
+		const u64 block = size - result;
```

Comment:
this obviously still has to be 
```suggestion
		const u64 block = std::min<u64>(size - result, local_buf.size());
```

Reply:
Fixed.